overlord/snapstate/snapmgr.go

```diff
@@ -373,8 +372,6 @@ func (m *SnapManager) doUnlinkSnap(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
-	// mark as inactive
-	snapst.Active = false
```

Comment:
of course then it means we need to capture a concept of previous active/current.

which relates to my final wondering, anyway not unsetting Current means that some places might try to do things with the snap thinking it's active which it isn't